wgsl/index.bs

```diff
@@ -9879,10 +9879,18 @@ For each [=formal parameter=] of a function, one or two tags are computed:
   <thead>
     <tr><th>Parameter Tag<th>Description
   </thead>
-  <tr algorithm="ParameterRequiredToBeUniform tag">
+  <tr algorithm="ParameterRequiredToBeUniformValue tag">
       <td><dfn noexport>ParameterRequiredToBeUniform.S</dfn>,<br>
           where |S| is one of the severities: [=severity/error=], [=severity/warning=], or [=severity/info=].
       <td>The parameter must be a [=uniform value=].
+          If the parameter type is a pointer, the memory view must be uniform.
+          Otherwise a diagnostic with severity |S| will be triggered.
+
+          Associated with a [=witness set=].
+  <tr algorithm="ParameterContentsRequiredToBeUniform tag">
+      <td><dfn noexport>ParameterContentsRequiredToBeUniform.S</dfn>,<br>
+          where |S| is one of the severities: [=severity/error=], [=severity/warning=], or [=severity/info=].
+      <td>The contents of the pointer parameter must be a [=uniform value=].
```

Comment:
Suggest: 

> The parameter is a pointer, and the contents of its memory view must be a [=uniform value=].

Or alternately mirror the description of PointerParameterMayBeNonUniform 

> The value stored in the memory pointed to by the pointer parameter must be a [=uniform value=].